tests/integration/test_logcollector/test_location_wildcards/data/test_cases/cases_location_wildcards.yaml

```diff
@@ -0,0 +1,47 @@
+- name: Test single asterisk wildcard
```

Comment:
We have to use names compatible with the `-k` parameter.

Reply:
Done in https://github.com/wazuh/wazuh-qa/pull/3906/commits/ffca56f7b9804b4ef1df2c7ffdd10987414e6e48